be/src/exec/spill/block_manager.h

```diff
@@ -53,8 +53,6 @@ class Block {
     bool is_remote() const { return _is_remote; }
     void set_is_remote(bool is_remote) { _is_remote = is_remote; }
 
-    virtual bool preallocate(size_t write_size) = 0;
```

Comment:
why remove this interface?

Reply:
after #50207 and #52020 this interface will never used.